tac/project-updates/2025/2025-Q2-Cacti.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: 2025 Q2 Cacti
+parent: 2025
+grand_parent: Project Updates
+---
+
+# Project Health
+
+- We are forging ahead with the IETF SATP reference implementations, the codebase integration and a mentorship project proposal which was granted acceptance just a few days ago.
+- We are still working on the Cacti Chainlink CCIP connector which we've presented in the February LFDT Interoperability Workshop mid-February this year.
+- We had an LFDT mentorship project approved for the summer, which will further the goal of creating an integrated architecture as envisioned in the project's roadmap.
+- A steady stream of questions are being answered on Discord where most of our activity is happening these days.
+- There haven't been any violations of the code of conduct by contributors so we count ourselves lucky so far.
+- There's been a bit of a slowdown this quarter, the number of active Contributors are decreased by 9.09%.
+
+# Questions/Issues for the TAC
+
+N/A
+
+# Releases
+
+December 2, 2024 - `v2.1.0`
```

Comment:
This feels a little old given that the report is a Q2 2025 report. It feels to me like it would be more accurate to says there have been no releases in Q1/Q2 2025.

Reply:
Right. I think we've just been using this format in past quarterly reports, and this is the first quarterly report this year (the first being the annual review), so we just made a note of the last release.

There were plans to release a new version (`3.0.0`) not too long ago, but Peter and the Accenture maintainers had to step away from the project, so that release is still up in the air. We'll try to resume the process soon, and discuss any issues we face in the next quarterly review.